src/app/qgisapp.cpp

```diff
@@ -16236,7 +16237,7 @@ void QgisApp::keyPressEvent( QKeyEvent *e )
 {
   emit keyPressed( e );
 
-#if 0 && defined(_MSC_VER) && defined(QGISDEBUG)
+#if 0 && defined(QGISDEBUG)
```

Comment:
What's this for? `#if 0` here disables this anyway?

Reply:
So I can change it easily to a 1 to test 😂

Reply:
So this was on purpose and now includes other platforms next to MSVC? :+1:

Reply:
Yes, before the crash dialog only worked on windows